<launch>
   <node name="image_publihser" pkg="image_publisher_cpp" type='image_publisher'>
      <param name="id" value="0" type="int" />    
      <param name="fps" value="30" type="int" />    
      <param name="calib_file_url" value="file:///home/rouholla/raspi-ir-tracker/catkin_ws/ost.yaml" type="string" />
      <remap from='/camera_publisher/camera_info' to='/camera/camera_info' />
      <remap from='/camera_publisher/raw_image' to='/camera/image_raw' />
   </node>
</launch>
